src/components/calendarInput/index.jsx

```diff
@@ -0,0 +1,112 @@
+import CalendarIcon from 'assets/calendar.png';
+import { addDays, addYears, endOfYear, format, startOfMonth } from 'date-fns';
+import React, { useEffect, useRef, useState } from 'react';
+import { DayPicker } from 'react-day-picker';
+import { useDispatch, useSelector } from 'react-redux';
+import { setDate } from 'store/slices/inputsSlice';
```

Comment:
минус сортировщика в еслинте, который ты поставил как раз тут отлично виден. Все импорты перемешаны и сортировка просто не имеет логики. Я стараюсь придерживаться такого формата:
внешние либы

внутренние файлы

относительный путь и прочее

Разделять пробелом логические части бывает удобно. Тут это будет выглядеть так:
```suggestion
import React, { useEffect, useRef, useState } from 'react';
import { useDispatch, useSelector } from 'react-redux';
import { DayPicker } from 'react-day-picker';
import { addDays, addYears, endOfYear, format, startOfMonth } from 'date-fns';

import CalendarIcon from 'assets/calendar.png';

import { setDate } from 'store/slices/inputsSlice';
```

Reply:
Done